Fabric.Authorization.API/Modules/UsersModule.cs

```diff
@@ -270,6 +303,58 @@ public class UsersModule : FabricModule<User>
                     HttpStatusCode.NotFound);
             }
         }
+
+        private async Task<dynamic> GetUserRoles()
+        {
+            this.RequiresClaims(AuthorizationReadClaim);
+            var roleUserRequest = this.Bind<RoleUserRequest>();
+            try
+            {

```

Comment:
Verify `subjectId` and `identityProvider` are not null?

Reply:
I'll add that verification.